src/layout/parser/mod.rs

```diff
@@ -779,6 +779,12 @@ fn parse_general_settings(layout: &mut Layout, reader: &mut Reader<'_>) -> Resul
         }),
         "ImageOpacity" => percentage(reader, |v| image_opacity = v),
         "ImageBlur" => percentage(reader, |v| image_blur = v),
+        "DropShadow" => {
+            parse_bool(reader, |b| settings.drop_shadow = b)
+        }
+        "ShadowColor" => color(reader, |color| {
+            settings.shadow_color = color;
+        }),
```

Comment:
These should be "DropShadows" and "ShadowsColor" respectively, at which point the parsing will actually start working and a lot of rendering tests will start rendering out the shadows properly (resulting in you needing to update their hashes).